src/makeRobot.js

```diff
@@ -38,7 +38,62 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version}, wheels: ${
+        this.wheels
+      }`;
+    },
+    get location() {
```

Comment:
referring to previous mentor's comment (add an empty line)
```suggestion
    },
    
    get location() {
```